source/jormungandr/jormungandr/realtime_schedule/synthese.py

```diff
@@ -222,3 +225,62 @@ def _timestamp_to_date(self, timestamp):
         dt = datetime.utcfromtimestamp(timestamp)
         dt = pytz.utc.localize(dt)
         return dt.astimezone(self.timezone)
+
+    def _find_route_point(self, route_point, passages):
+        """
+        To find the right passage in synthese:
+
+        As a reminder we query synthese only for a stoppoint and we get, for all the routes that pass by
+        this stop, the next passages.
+        The tricky part is to find the which route concerns our routepoint
+
+         * we first look if by miracle we can find a route with the synthese code of our route in it's
+         external codes (it can have several if the route is a fusion of many routes)
+            -> if we found the route, we return it's passages
+         * else we query navitia to get all routes that pass by the stoppoint for the line of the route point
+            * if we get only one route, we search for this route's line in the synthese response
+                (because lines synthese code are move coherent)
+                -> we concatenate all synthese passages on this line
+         -> else we return the base schedule
+        """
+        log = logging.getLogger(__name__)
+        stop_point_id = str(route_point.fetch_stop_id(self.object_id_tag))
+        is_same_route = lambda syn_rp: syn_rp.syn_route_id in route_point.fetch_all_route_id(self.object_id_tag)
+        p = next((p for syn_rp, p in passages.items()
```

Comment:
We only return the passages of one route point here no?
Since route have been merge we need to return the passages of all the synthése's routes of the navitia route